docgen/src/guide/Server-side_rendering.md

```diff
@@ -6,8 +6,67 @@ category: guide
 navWeight: 30
 ---
 
-React InstantSearch is not yet compatible with server-side rendering. This is one of our priority topics
-right now. We will soon provide API entries to handle this usecase.
+React InstantSearch is compatible with server-side rendering. 
+
+We provide two API entries to handle it: 
+
+* a new type of [`<InstantSearch>`](widgets/InstantSearch.html) available under `'react-instantsearch/server'`
+* a `findResultsState` function available under `'react-instantsearch/server'`
+
+Here are the steps you need to follow to migrate:
+
+1. Import [`<InstantSearch>`](widgets/InstantSearch.html) from `'react-instantsearch/server'` instead of `'react-instantsearch/dom'`. 
+
+2. Use [`findResultsState`](server-side-renderings/findResultsState.html) to get a `resultsState` server side. 
+
+```jsx
+server.get('/', async (req, res) => {
+  const resultsState = await findResultsState(App);
+
+  [...]
+});
+```
+
+**(Optional)**: If you need to forward a `searchState` to your `<InstantSearch/>` instance, just pass it as a second parameter of the findResultsState function. 
+
+```jsx
+server.get('/', async (req, res) => {
+  const resultsState = await findResultsState(App, {searchState});
+
+  //[...]
+});
+```
+
+3. Forward this `resultState` as a prop of your `<InstantSearch>` instance. 
+
+```jsx
```

Comment:
App change should come first before findResult usage. It must also include the new import statement